common/templates/options.php

```diff
@@ -203,6 +201,23 @@
 </form>
 <script>
     (function ($) {
+        $.get("admin-ajax.php",
+            {
+                action: <?= json_encode($plugin->woocommerce->getProductKeysAction()); ?>,
+                selected_key: <?= json_encode($config["custom_gtin"]); ?>,
+            }
```

Comment:
Encode the entire object with `json_encode`. It's shorter and avoids formatting accidents (missing commas, etc.).